setup.py

```diff
@@ -98,7 +98,7 @@ def run(self):
       include_package_data=True,
       packages=find_packages(),
       install_requires=[
-          'django>=1.8,<2.1',
+          'django>=2.1,<3.0',
```

Comment:
How do we feel about just pinning to 2.2.x, since it's the LTS version?

Reply:
seems reasonable to me